<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Калькулятор решения СЛАУ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Количество переменных:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nLineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="inputButton">
       <property name="text">
        <string>Ввести</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Коэффициенты перед переменными</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Свободные коэффициенты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solveButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>87</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Решить</string>
    </property>
   </widget>
   <widget class="QLabel" name="xEqualsView">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X=</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="variablesTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>361</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="freeOddsTable">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>101</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="solveTable">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Метод обратной матрицы</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Формулы Крамера</string>
    </property>
   </widget>
   <widget class="QLabel" name="xEqualsView_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>370</y>
      <width>41</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X=</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="solveTableKramer">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>310</y>
      <width>121</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="detLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>421</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Определитель основной матрицы системы:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
